client/commands/v2/pysa_server.py

```diff
@@ -60,6 +65,77 @@ def __init__(
         self.binary_location = binary_location
         self.server_identifier = server_identifier
 
+    def invalid_model_to_diagnostic(
```

Comment:
`invalid_model_to_diagnostic`, `invalid_models_to_diagnostics`, and `update_errors` are present because you're based off of Sarthak's branch, right? You didn't change these?

Reply:
Yes, these were pulled from Sarthak's branch!